x-pack/test/accessibility/apps/ml.ts

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import path from 'path';
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
```

Comment:
It seems `getPageObjects` is not used, so could be removed here.

Reply:
Done in https://github.com/elastic/kibana/pull/88323/commits/55ed7968b20327dd9d5205bfb6c89d877aa3fd92